lib/engine/game/base.rb

```diff
@@ -2775,6 +2775,18 @@ def train_power?
       def show_map_legend?
         false
       end
+
+      # This is a bit of a hack to get around how 1840 and 18USA have different and incompatible opinions on what the
+      #  option to use the scrap_trains action in a merger round means
+      # If a game overrides this to true, then if the possible actions for the current entity include any of:
+      # * buy_train
+      # * scrap_train
+      # * reassign_train
+      # then the Operating view will be used instead of the Merger round view in a merger round.
+      # See https://github.com/tobymao/18xx/issues/7169
```

Comment:
I don't know if the history is important. Maybe it could be simplified to something like "Renders the operating round view for train actions, even when in another round type."

Reply:
I trimmed down the comment